examples/pytorch/multiple-choice/run_swag.py

```diff
@@ -323,7 +324,7 @@ def main():
             )
         max_seq_length = min(data_args.max_seq_length, tokenizer.model_max_length)
 
-    # Preprocessing the datasets.
+    # Preprocessing the raw_datasets.
```

Comment:
perhaps best to say "Preprocessing raw_datasets" or leave it as "Preprocessing the datasets"

Reply:
I left it as it is